### PR TITLE
Bonnie 1513

### PR DESCRIPTION
**Pull request by mayerm94**
Change attribute status to qdm_status, add tests for method and status to substance and labtestperformed

Pull requests into CQL QDM Patient API require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] This PR is into the correct branch.
- [x] JIRA ticket for this PR: https://jira.mitre.org/browse/BONNIE-1513
- [x] JIRA ticket links to this PR
- [x] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [x] Code coverage has not gone down and all code touched or added is covered.

Branch | Coverage
-- | --
master | ================ Coverage summary ================<br>Statements   : 74.25% ( 1629/2194 )<br>Branches     : 53.76% ( 415/772 )<br>Functions    : 64.96% ( 317/488 )<br>Lines        : 72.35% ( 1405/1942 )<br>===============================================
bonnie_1513  | ================ Coverage summary ================<br>Statements   : 74.41% ( 1640/2204 )<br>Branches     : 54.64% ( 424/776 )<br>Functions    : 64.69% ( 317/490 )<br>Lines        : 72.54% ( 1416/1952 )<br>===============================================
- [x] Automated regression test(s) pass

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA test links:
- [x] Justification for using JIRA tests:
- [x] JIRA tests have been added to sprint

**Reviewer 1:**

Name: @c-monkey
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass: N/A
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree: N/A

**Reviewer 2:**

Name: @hossenlopp 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests: N/A
- [x] JIRA tests have been run and pass N/A
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree N/A

### CODE REVIEW

File: app/assets/javascripts/datatypes/laboratorytest.js.coffee [hunk 1]
```diff
@@ -19,6 +19,7 @@ class CQL_QDM.LaboratoryTestOrder extends CQL_QDM.QDMDatatype
     @_negationRationale = @entry.negationReason
     @_reason = @entry.reason
     @_authorDatetime = CQL_QDM.Helpers.convertDateTime(@entry.start_time)
+    @_status = @entry.qdm_status
```
Comment (c-monkey) [inline app/assets/javascripts/datatypes/laboratorytest.js.coffee]: this line is not covered in tests

File: app/assets/javascripts/datatypes/laboratorytest.js.coffee [hunk 2]
```diff
@@ -53,6 +54,15 @@ class CQL_QDM.LaboratoryTestOrder extends CQL_QDM.QDMDatatype
   authorDatetime: ->
     @_authorDatetime
 
+  ###
```
Comment (c-monkey) [inline app/assets/javascripts/datatypes/laboratorytest.js.coffee]: this function is not covered in tests

File: app/assets/javascripts/datatypes/laboratorytest.js.coffee [hunk 3]
```diff
@@ -208,6 +218,7 @@ class CQL_QDM.LaboratoryTestRecommended extends CQL_QDM.QDMDatatype
     @_method = @entry.method
     @_negationRationale = @entry.negationReason
     @_reason = @entry.reason
+    @_status = @entry.qdm_status
```
Comment (c-monkey) [inline app/assets/javascripts/datatypes/laboratorytest.js.coffee]: not covered

File: app/assets/javascripts/datatypes/laboratorytest.js.coffee [hunk 4]
```diff
@@ -241,3 +252,12 @@ class CQL_QDM.LaboratoryTestRecommended extends CQL_QDM.QDMDatatype
       new cql.Code(@_reason.code, @_reason.code_system)
     else
       null
+
```
Comment (c-monkey) [inline app/assets/javascripts/datatypes/laboratorytest.js.coffee]: not covered

File: app/assets/javascripts/datatypes/laboratorytest.js.coffee (hunk 1 above)
Comment (mayerm94) [reply app/assets/javascripts/datatypes/laboratorytest.js.coffee]: That's not possible for this one line in the constructor to not be hit, and I am not seeing that when I open the coverage results. Are you sure you ran `bundle exec rake teaspoon USE_COVERAGE=default` ?

Comment (hossenlopp) [reply app/assets/javascripts/datatypes/laboratorytest.js.coffee]: I concur with @c-monkey, I'm not seeing this constructor hit at all.
screen shot 2018-06-04 at 8 22 48 am

Comment (mayerm94) [reply app/assets/javascripts/datatypes/laboratorytest.js.coffee]: I forgot that I had un-pushed changes with tests added for LaboratoryTestOrder

File: app/assets/javascripts/datatypes/laboratorytest.js.coffee (hunk 3 above)
Comment (c-monkey) [reply app/assets/javascripts/datatypes/laboratorytest.js.coffee]: The above coverage issue have been resolved, but LaboratoryTestRecommended is still uncovered
